src/main/webapp/map.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Map Project</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCII0kJqaa9uBysCMxSVcYJn1d2KeuAcDU"></script>
+  <script>
+    let map;
+    function createMap(){
+      map = new google.maps.Map(document.getElementById('map'), {
+        center: {lat: 37.422, lng: -122.084},
+        zoom: 16
+      });
+    }
+  </script>
+  <style>
```

Comment:
Same here, move this to a separate file. Look at user-page.html for an example on how this is done.

Reply:
Moved the javascript to a separate file